pilot/pkg/config/kube/crdclient/client_test.go

```diff
@@ -125,7 +129,7 @@ func TestClient(t *testing.T) {
 
 			pb, err := r.NewInstance()
 			if err != nil {
-				t.Fatal(err)
+				t.Error(err)
```

Comment:
nit: why the change? Fatal short circuits, I think the rest of the test is not valid if this fails